packages/back-end/types/experiment-snapshot.d.ts

```diff
@@ -14,6 +17,7 @@ export interface SnapshotMetric {
   risk?: [number, number];
   stats?: MetricStats;
   pValue?: number;
+  pValueAdjusted?: number;
```

Comment:
This should persist even going forward since at some point we will want both pvalues side-by-side even if they both come from the back-end.